buildCommands.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+i=0
+
+docker_apps=""
+
+for docker_app in $docker_apps; do
+    i=$((i+1))
+    BUILD_APPS[$i]=$docker_app
+    BUILD_CMD[$i]="( cd $docker_app && ./buildDocker.sh )"
+done
+
+maven_apps="regnskapsregister"
+
+for maven_app in $maven_apps; do
```

Comment:
Good to have build-script support for this, but are we ever going to have multiple maven apps in this project?